configure.ac

```diff
@@ -6,6 +6,7 @@ AC_INIT([Heimdal],[1.6.99],[heimdal-bugs@h5l.org])
 AC_CONFIG_SRCDIR([kuser/kinit.c])
 AC_CONFIG_HEADERS(include/config.h)
 AC_CONFIG_MACRO_DIR([cf])
+AC_USE_SYSTEM_EXTENSIONS
```

Comment:
I'm not sure this is a good idea.  We should know where and why we need extensions rather than enable them globally, otherwise we risk breaking builds more often.  Mind you, we can always fix them after the fact.  But I'd rather that developers trip over the need to enable extensions and then do whatever the right thing is, such as adding compatibility wrappers in libroken.